airflow/www/views.py

```diff
@@ -563,7 +563,9 @@ def last_dagruns(self, session=None):
             return wwwutils.json_response({})
 
         query = session.query(
-            DagRun.dag_id, sqla.func.max(DagRun.execution_date).label('last_run')
+            DagRun.dag_id,
+            sqla.func.max(DagRun.execution_date).label('execution_date'),
+            sqla.func.max(DagRun.start_date).label('start_date'),
```

Comment:
This could be incorrect an some (unlikely) circumstance:

If I create an dag run for an older execution_date, this would show mixed values.

Unfortunately doing this correctly in a single SQL query is harder.

Reply:
I can test for that, thanks for the heads up. However I would think the value above it, the execution_date, would be wrong to get the max value in the query, not the run_date. I don't recall that being the case as that query has been the same for a long time now (I just renamed the column alais) but if it is wrong, I'll try and fix it.

Reply:
Sorry, got busy with other stuff, I'll look into this soon

Reply:
Fair point -- if you could rebase this PR we can merge this as is, as this is right in most cases, and better than the current broken behavior.

Reply:
rebased now